src/static/css/main.css

```diff
@@ -6,6 +6,15 @@
     width: 100%;
     text-align: center;
 }
+#controls {
+  -moz-column-count: 2;
+  -webkit-column-count: 2;
+  column-count: 2;
+}
+#plop {
+  position: absolute !important;
+  align: center;
+}
```

Comment:
fit as zoom and pan only works with fixed size viewport.
we have a floating viewport size in index.html so this should be probably applied to another page.

Reply:
Fit works fine with a floating viewport: 
![image](https://user-images.githubusercontent.com/4921914/96693207-3b387300-1387-11eb-896c-b12b66c42850.png)